src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -188,10 +188,13 @@ export default defineComponent({
         : undefined
 
     const updateTimeLoop = () => {
-      if (localAudio && status.value === 'playing') {
-        currentTime.value = localAudio.currentTime
-        window.requestAnimationFrame(updateTimeLoop)
-      }
+      if (localAudio)
+        if (status.value === 'playing') {
+          currentTime.value = localAudio.currentTime
+          window.requestAnimationFrame(updateTimeLoop)
+        } else if (status.value === 'played') {
+          currentTime.value = localAudio.currentTime
+        }
```

Comment:
I'm going to look into why later but I wanted to note it here so I don't forget. It looks like the final update to put the audio current time happens slightly later and causes a jump in the waveform progression at the end for short audio files :thinking: I want to try to find a fix for that too if you don't mind @obulat?

Reply:
I wonder if it was caused by the audio sometimes ending with a `paused` status, and sometimes with `played`. I still don't understand why it happens, but I removed the check for `status` being played to update the `currentTime` on line 196 here.